$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_doc_gcov_1,v 1.1 2009/12/06 15:52:19 tg Exp $
--- gcc/doc/gcov.1.orig	Tue Mar 13 09:47:07 2012
+++ gcc/doc/gcov.1	Sun Dec  1 04:10:35 2013
@@ -123,17 +123,17 @@
 .rm #[ #] #H #V #F C
 .\" ========================================================================
 .\"
-.IX Title "GCOV 1"
-.TH GCOV 1 "2012-03-13" "gcc-4.4.7" "GNU"
+.IX Title "GCOV-VSS 1"
+.TH GCOV\-VSS 1 "2012-03-13" "gcc-4.4.7" "GNU"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
 .nh
 .SH "NAME"
-gcov \- coverage testing tool
+gcov\-VSS \- coverage testing tool
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
-gcov [\fB\-v\fR|\fB\-\-version\fR] [\fB\-h\fR|\fB\-\-help\fR]
+gcov\-VSS [\fB\-v\fR|\fB\-\-version\fR] [\fB\-h\fR|\fB\-\-help\fR]
      [\fB\-a\fR|\fB\-\-all\-blocks\fR]
      [\fB\-b\fR|\fB\-\-branch\-probabilities\fR]
      [\fB\-c\fR|\fB\-\-branch\-counts\fR]
